developers.md

```diff
@@ -1,28 +1,82 @@
 # rules_pkg - Information for Developers
 
-NOTE: This is a work in progress. It is only an outline at this point.
+NOTE: This is perpetually a work in progress. We will revise it as we need.
 
-TBD: Introduction
+## PR guidelines
 
-## General
+Please discuss proposed major features and structural changes before sending a
+PR. The best way to do that is to
 
--    tests
--    OS portability
--    minimize tests
--    repo structure
-     - more at build time than in the distribution
-     - some features do not work on all platforms
+1.  Create an an issue
+1.  Discuss it in the issue or in rules-pkg-discuss@googlegroups.com
+1.  Bring it up at the monthly engineering meeting. (See #325 for
+    details)[https://github.com/bazelbuild/rules_pkg/issues/325].
 
-## Starlark style
+### A few small PRs are better than one big one.
 
--    using analysis tests
--    docstrings in attributes
+If you need to refactor a lot of code before you can add new behavior, please
+send a refactoring CL first (which should not add or change tests), then send a
```

Comment:
"PR" is better than "CL" when you're in github territory.  CL is very much a perforce-specific term that many people might not know about.

Reply:
Yes